### PR TITLE
feature#182201681-implement requests table

### PR DESCRIPTION
**Pull request by Patricknshimiyimana**
#### What does this PR do?
 - Implement user travel requests table
#### Description of Task to be completed?
 - create travel requests component
 - create table using mui
#### How should this be manually tested?
 - clone this repository
 - checkout to this branch `ft-requests-table-182201681`
 - run `npm install` to install dependencies
 - run `npm start` to start the application

#### Any background context you want to provide?
 - I used mui because it is the best, clean and easier implementation 
  to display requests in a table, I could find online
#### What are the relevant pivotal tracker stories?
 - https://www.pivotaltracker.com/n/projects/2569105/stories/182201681
#### Screenshots(If appropriate)
Screen Shot 2022-08-18 at 09 05 09

### CODE REVIEW

File: src/components/dashboard/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  const { window } = props;
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
+
+  const drawer = (
+    <div>
+      {/* <Toolbar /> */}
```
Comment (semy-tuyi) [inline src/components/dashboard/Dashboard.js]: remove this comment

**Review by semy-tuyi — CHANGES_REQUESTED** [review]
Improve coverage

**Comment by Patricknshimiyimana** [comment]
> Improve coverage

Working on that
Thanks.

**Review by alaintresor — APPROVED** [review]

**Review by uwambajeddy — CHANGES_REQUESTED** [review]
Even though you had no figma UI to reference on but atleast there some features that are supposed to look similar 
Screenshot (188)

**Comment by Patricknshimiyimana** [comment]
> Even though you had no figma UI to reference on but atleast there some features that are supposed to look similar !

Hello @uwambajeddy please note that we are using a mui table and I'm using the dashboard @mwisemarierose designed so that we have the same dashboard

**Review by joshwambere — APPROVED** [review]

**Pull request merged**